.github/workflows/neurips23.yml

```diff
@@ -28,8 +28,8 @@ jobs:
             dataset: random-xs
             track: ood
           - algorithm: cufe
-            dataset: sparse-small
-            track: sparse
+            dataset: random-s
+            track: filter
```

Comment:
are you replacing the sparse CI with the filter CI here? Please fix.

Reply:
I just fixed it